tests/core/actions/test_forms.py

```diff
@@ -34,38 +35,69 @@
 from rasa.shared.core.trackers import DialogueStateTracker
 from rasa.utils.endpoints import EndpointConfig
 
+ACTION_SERVER_URL = "http://my-action-server:5055/webhook"
+
 
 async def test_activate():
```

Comment:
I'd actually recommend keeping this unit test unchanged, and then writing a new test with the custom slot as you've written here.